bundles/org.openhab.binding.bluetooth.daikinmadoka/src/main/java/org/openhab/binding/bluetooth/daikinmadoka/handler/DaikinMadokaHandler.java

```diff
@@ -121,7 +129,30 @@ public void initialize() {
             submitCommand(new GetPowerstateCommand()); // always keep the "GetPowerState" aftern the "GetOperationMode"
             submitCommand(new GetSetpointCommand());
             submitCommand(new GetFanspeedCommand());
-        }, 10, c.refreshInterval, TimeUnit.SECONDS);
+            submitCommand(new GetCleanFilterIndicatorCommand());
+
+            // As it is a complex operation - it has been extracted to a method.
+            retrieveOperationHours();
+
+            submitCommand(new GetEyeBrightnessCommand());
+        }, new Random().nextInt(30), c.refreshInterval, TimeUnit.SECONDS); // We introduce a random start time, it
+                                                                           // avoids when having multiple devices to
+                                                                           // have the commands sent simultaneously.
+    }
+
+    private void retrieveOperationHours() {
+        // This one is special - and MUST be ran twice, after being in priv mode
+        // run it once an hour is sufficient... TODO
+        submitCommand(new EnterPrivilegedModeCommand());
+        submitCommand(new GetOperationHoursCommand());
+        // a 1second+ delay is necessary
+        try {
+            Thread.sleep(1500);
+        } catch (InterruptedException e) {
+            logger.error("Error in retrieveOperationHours()", e);
```

Comment:
The error logging level should be reserved for issues catastrophic enough to threaten the operation of openHab itself. An error in a binding shouldn't qualify for this. Please read the [openHab coding guidelines](https://www.openhab.org/docs/developer/guidelines.html#f-logging) for expected use of logging in bindings.